### PR TITLE
Please check my Session # 7 homework

### PR DESCRIPTION
**Pull request by natashashvakel**
Ellie, please review my homework for Session 7

### CODE REVIEW

File: tests/natasha_homework_7.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        self.enter_employee_ID("441915")
+
+        self.browser.find_element_by_id('searchBtn').click()
+
+        time.sleep(1)
+
+        result = self.browser.find_element_by_xpath('//tbody/tr[1]/td[2]').text
+        self.assertEqual('441915', result)
+
+        list_of_web_elements_IDs = self.browser.find_elements_by_xpath('//*[@id="resultTable"]/tbody/tr/td[5]')
```
Comment (EllieSky) [inline tests/natasha_homework_7.py:49]: You are finding the fifth column... why?
IDs are in column 2... or you can just count rows, don’t need to include td at all.

File: tests/natasha_homework_7.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        time.sleep(1)
+
+        list_of_web_elements_employee_names = self.browser.find_elements_by_xpath("//tbody/tr/td[3]")
+
+        count = 0
+        for item in list_of_web_elements_employee_names:
+            if item.text.find('David') != -1:
+                count = count + 1
+
+        self.assertEqual(count, len(list_of_web_elements_employee_names), "Not all entries contain David")
```
Comment (EllieSky) [inline tests/natasha_homework_7.py:69]: Unique approach, but it works.

File: tests/natasha_homework_7.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        self.enter_employee_name("amb")
+        self.browser.find_element_by_id('searchBtn').click()
+
+        time.sleep(1)
+
+        list_of_web_elements_employee_names = self.browser.find_elements_by_xpath("//tbody/tr/td[3]")
+        list_of_web_elements_employee_last_names = self.browser.find_elements_by_xpath("//tbody/tr/td[4]")
+
+        count = 0
+        for i in range(len(list_of_web_elements_employee_names)):
```
Comment (EllieSky) [inline tests/natasha_homework_7.py:83]: Very nice, you got it!

**Review by EllieSky — COMMENTED** [review]
Good quality work! I especially enjoyed that you took the initiative to add additional steps for entering the name and ID. 

Consider including an assertion in your test to verify that sending the escape key actually closes the auto complete element.

File: tests/natasha_homework_7.py (hunk 1 above)
Comment (natashashvakel) [reply tests/natasha_homework_7.py:49]: Got it, it should have been second column (copy/paste error). I understand now I could use the //*[@id="resultTable"]/tbody/tr

Thank you for review.

**Comment by natashashvakel** [comment]
closing

**Pull request closed without merging**